lib/perl/Genome/Model/ClinSeq/Command/UpdateAnalysis.pm

```diff
@@ -194,6 +194,15 @@ class Genome::Model::ClinSeq::Command::UpdateAnalysis {
               is => 'Boolean',
               doc => 'Allow imported instrument data to be included',
         },
+        validation_as_exome => {
+              is => 'Boolean',
+              doc => 'Treat validation data as exome data. Includes instrument data from validation capture in exome reference alignment model.',
+        },
+        my_trsn => {
+              is => 'Text',
+              doc => 'If using validation-as-exome option, option to specify what instrument data to derive the target-set-region-name and region-of-interest from. valid options: exome or validation', 
+              valid_values => ['exome','validation']
+        },
```

Comment:
Someone else may know better, but I believe the way to accomplish that would be to make it an optional parameter with `valid_values` of `exome` and `validation`. That would allow for it to be either exome, validation, or undef. It does seem to appear that this parameter is currently optional, so I think it could be used in that manner already.